components/src/main/java/org/wso2/carbon/messaging/websocket/WebSocketCarbonMessage.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *   Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *   WSO2 Inc. licenses this file to you under the Apache License,
+ *   Version 2.0 (the "License"); you may not use this file except
+ *   in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.wso2.carbon.messaging.websocket;
+
+import org.wso2.carbon.messaging.CarbonMessage;
+
+/**
+ * {@link CarbonMessage} type of WebSocket Messages
+ */
+public class WebSocketCarbonMessage extends CarbonMessage {
```

Comment:
Yes. It seems that is good to use WebSocketMessage instead of WebSocketCarbonMessage and the same naming convention for all other extended WebSocketMessages. 
Thanks.